src/ops/cloud/VariablesOps.ts

```diff
@@ -226,6 +264,40 @@ export async function readVariables({
   return (await _getVariables({ state })).result;
 }
 
+export async function exportVariable({
+  variableId,
+  state,
+}: {
+  variableId: string;
+  state: State;
+}): Promise<VariablesExportInterface> {
+  debugMessage({ message: `VariablesOps.exportVariable: start`, state });
+  const exportData = createVariablesExportTemplate({ state });
+  const variable = await readVariable({ variableId, state });
+  variable.value = decode(variable.valueBase64);
```

Comment:
Is there any reason to leave the variable encoded unless specified via an option (e.g. --decode)?

Reply:
I would say the default should be to decode the value and have base64 encoding as an option. Frodo's other Variable commands expect the value to not be encoded and already encode the value passed to it. For example: [VariablesApi.putVariable](https://github.com/trivir/frodo-lib/blob/1b2a449e7c4c4611b0707a2dc6d6dc592748082d/src/api/cloud/VariablesApi.ts#L132).